crates/miden-objects/src/account/mod.rs

```diff
@@ -410,6 +417,80 @@ impl Account {
     }
 }
 
+impl TryFrom<Account> for AccountDelta {
```

Comment:
This seems to only be used in tests and testing env (`MockChainBuilder::build`) - should we place this behind a `#[cfg(any(feature = "testing", test))]`?

Reply:
> I don't quite recall what we discussed specifically

Me neither. Unfortunately, I can't find the discussion anymore.

I would move this impl behind `testing` for now as @mmagician suggested and we can easily expose this if it becomes useful for the node when you revisit the topic.